layouts/index.html

```diff
@@ -67,6 +67,12 @@ <h2 class="title--answer">Mostly...</h2>
         <a href="https://gitlab.gnome.org/GNOME/eog">Eye of GNOME</a>,
         <a href="https://github.com/eXeC64/imv">imv</a>
       </li>
+      <li class="list__item--ok">
+        Login manager:
+        <a href="https://github.com/max-moser/lightdm-elephant-greeter">Elephant-greeter</a>,
```

Comment:
should we rename to "LightDM Elephant Greeter"? it seems to be the official name on their GH

Reply:
I was unsure about that too. But I suppose since it is a greeter for LightDM and LightDM is a dependency it would make more sense to call it that. I'll make the change.